masterlist.yaml

```
@@ -4364,6 +4364,36 @@ plugins:
       - crc: 0x7287565E
         util: 'TES4Edit v4.0.4'
 
+  - name: '((Better (Baurus|Martin|Uriel Septim)|Improved NPC Faces|Vanilla Kalthar Face) For OCOv2(.*))\.esp'
```

There are some unnecessary capture groups in this regex. The archived file can be removed from it too:

```suggestion
  - name: '(Better (Baurus|Martin|Uriel Septim)|Improved NPC Faces) For OCOv2.*\.esp'
```